test/driver.ts

```diff
@@ -101,6 +113,8 @@ export class Driver {
         await contractRegistry.set("compliance", compliance.address);
         await contractRegistry.set("validatorsRegistration", validatorsRegistration.address);
         await contractRegistry.set("committee", committee.address);
+        await contractRegistry.set("_bootstrapToken", externalToken.address);
+        await contractRegistry.set("_erc20", erc20.address);
```

Comment:
The contracts don't use the contract registry to find the token contracts, they get them in the constructor. I understand why you added them, but it's a bit misleading.

Reply:
the underscore is supposed to serve as a hint. I can make the hint more conspicuous:
`TESTKIT_ONLY_bootstrapToken`. What do you think?

There are two alternatives - one is to persist the driver state in a file.. I'd like to avoid that because it's messy on the user side. The other alternative is to use a second instance of the registry contract which is there for the driver to load from and not known to the contracts at all. I like that option but would like to defer that to an issue.